src/_tests/fixtures/44282/derived.json

```diff
@@ -2,36 +2,7 @@
   "type": "info",
   "pr_number": 44282,
   "author": "fishcharlie",
-  "owners": [
-    "CaselIT",
-    "alanmarcell",
-    "dante-101",
-    "mcortesi",
-    "EnricoPicci",
-    "AJCStriker",
-    "julien-c",
-    "daprahamian",
-    "denys-bushulyak",
-    "BastienAr",
-    "sindbach",
-    "geraldinelemeur",
-    "various89",
-    "angela-1",
-    "lirbank",
-    "hector7",
-    "floric",
-    "erikc5000",
-    "Manc",
-    "jloveridge",
-    "ranguna",
-    "HosseinAgha",
-    "albertossilva",
-    "peterblazejewicz",
-    "LinusU",
-    "taxilian",
-    "xamgore",
-    "avaly"
-  ],
+  "owners": [],
```

Comment:
Maybe suspicious?

Reply:
I think it’s right: https://github.com/DefinitelyTyped/DefinitelyTyped/pull/44282/files#diff-0870154c331487fa03d8b4284eb0583bL3

Not sure where the old list came from